src/setup/local_install_skaffold.md

```diff
@@ -119,30 +119,10 @@ sudo apt-get install -y kubectl=1.16.7-00 -V
 
 #### [Helm](https://helm.sh/)
 
-Please use [Helm v2.16.1](https://github.com/helm/helm/releases/tag/v2.16.1) to get Helm and Tiller for setups using `substra 0.6.0` or lower (see [compatibility table](https://github.com/SubstraFoundation/substra#compatibility-table)). Following releases should work fine with Helm v3 (which has removed Tiller).
-
-- Mac
-
-```sh
-# Helm v2
-brew install helm@2
-```
-
-- Ubuntu
-
-```sh
-# Helm v2
-# Get the executable
-curl -LO https://get.helm.sh/helm-v2.16.1-linux-amd64.tar.gz
-# Extract the downloaded archive
-tar xzvf helm-v2.16.1-linux-amd64.tar.gz
-cd linux-amd64/
-# Move the executables to your local bin
```

Comment:
Agreed! What do you think about letting the installation part (updated for helm3) for ubuntu and mac?

Reply:
There is already a [link to the helm doc](https://github.com/SubstraFoundation/substra-documentation/pull/141/files#diff-61aebd673ab4b235097ab2d17ea1045a9580c560ab0013d575bbcb34b0a1786fR122) for that.
I'm not sure that duplicating helm doc would make it clearer for the user. Also which installation way should we promote (wget the bin, apt repo, snapcraft, etc)?
Since the installation of helm is pretty standard, I would just rely on official doc.

Reply:
Okay then! My concern was about less experienced users, but you are right, as the link is provided this is fine!